### PR TITLE
fix(#3629): fixed broken `@as` binding

### PR DESCRIPTION
**Pull request by maxonfjvipon**
Closes: #3629

### CODE REVIEW

**Comment by maxonfjvipon** [comment]
@yegor256 please check

File: .github/workflows/mvn.yml [hunk 1]
```diff
@@ -73,4 +73,4 @@ jobs:
           echo multiplexing = false >> %USERPROFILE%\.cargo\config.toml
         shell: cmd
       - run: cargo --version
-      - run: mvn clean install -Pqulice --errors --batch-mode
+      - run: mvn clean install --errors --batch-mode
```
Comment (yegor256) [inline .github/workflows/mvn.yml:76]: @maxonfjvipon why?? :)

**Review by yegor256 — CHANGES_REQUESTED** [review]

**Comment by maxonfjvipon** [comment]
@yegor256 please check

**Comment by yegor256** [comment]
@maxonfjvipon thanks!

**Comment by 0crat** [comment]
@maxonfjvipon Hey there! 👋 Great job on your contribution! You've earned +4 points, which breaks down like this: +4 base, -16 for no code review (ouch!), -4 for only 4 comments, and +20 to keep you motivated. Remember, code reviews and comments are super important – they can boost your points big time! Next time, aim for at least 6 comments and don't forget that code review. Keep those [hits-of-code](https://www.yegor256.com/2014/11/14/hits-of-code.html) coming, but watch out for the 200 and 800 thresholds. [Keep up](https://www.yegor256.com/2018/03/06/speed-vs-quality.html) the good work! Your balance is now [+65](https://www.eolang.org/zerocracy/objectionary-vitals.html). 💪

**Pull request merged**